features/facebookImport/src/model/analyses/advertising-value-analysis.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import RootAnalysis from "./root-analysis.js";
+import i18n from "../../i18n.js";
+
+import AdvertisingValueMiniStory from "../../components/advertisingValueMiniStory/advertisingValueMiniStory.jsx";
+
+export default class AdvertisingValueAnalysis extends RootAnalysis {
+    get title() {
+        return i18n.t("advertisingValueMiniStory:title");
+    }
+
+    async analyze({ facebookAccount }) {
+        const randomAdInterests = [];
+        const numberInterests = facebookAccount.adInterests.length;
+        this.active = false;
+        if (numberInterests > 0) {
+            for (let i = 0; i < Math.min(3, numberInterests); i++) {
+                const randomIndex = parseInt(
```

Comment:
Not for now, but I think in an array with a few elements, this has a high chance of selecting the same element twice.

Reply:
Right. Non-zero chance in any case.

Reply:
Which would be fine, wouldn't it? I mean it's more like an easter egg or something 😅 
Out of curiosity though: How would you improve the randomization here?

Reply:
Current solution is definitely fine for now. To avoid any duplication we could generate a set of random unique indexes from the interval `[0, numberInterests.length-1]`  (so generate random indexes that are not already selected in a loop). And the return the elements at those indexes.